studiengaenge/lehramt_info.tex

```diff
@@ -9,10 +9,41 @@
+
+Im dritten Semester wird empfohlen die Mathematik-Veranstaltung, egal ob in Informatik oder im anderen Hauptfach, zu belegen. Diese Veranstaltung kann beispielsweise „Mathematik für Informatiker 1“(MafIn 1) sein.
+
+Ihr merkt schon, wir empfehlen ganz schön viel, da, wie bereits im allgemeinen Teil zum Lehramt gesagt, euch beim Zusammenstellen des Stundenplans ein Tauschen der Veranstaltungen nicht verwehrt bleibt. Dabei empfehlen wir im Stundenplan zwei Informatik-Veranstaltungen pro Semester zu besuchen. Dabei kann im Wintersemester zu den oben bereits genannten Veranstaltungen folgende Pflichtmodule gehört werden: 
+
+ „Einführung in die Technische Informatik“ (ITE oder Technische Info)\footnote{dafür hat leider noch niemand eine schön Abkürzung gefunden}
+ 
+„Software Engineering“ (ISW)
+
+Im Sommersemester müssen folgende Module belegt werden:
+
+„Einführung in die Theoretische Informatik“\footnote{„Theo“ kann zu Verwirrungen mit Physikern führen, da diese das als „Theoretische Physik“ verstehen}. Bei diesem Modul ist es sinnvoll die Mathematik-Vorlesung schon gehört zu haben.
+ 
+ 
+„Betriebssysteme und Netzwerke“ (BeNe)
+
+„Datenbanken 1“ (IDB1)
+
+Außerdem muss ein Seminar besucht werden, diese werden jedes Semester angeboten. 
+
+Studiert ihr den 50\%-Bachelor mit Lehramtsoption, dann müsst ihr noch ein Proseminar im Bereich Informatik und Gesellschaft belegen. Des Weiteren müsst ihr Fachdidaktik1 Teil 2 belegen. 
+
+Studiert ihr den 50\%-Bachelor ohne Lehramtsoption, dann müsst ihr ein Proseminar und ein Anfängerpraktikum, welche 6 Leistungspunkte „Fächerübergreifende Kompetenzen“ (FÜK) beinhalten, belegen. 
+
+\subsection{Hinweise zu Klausuren}
+In Informatik solltet ihr euch bei jeder Veranstaltung genau darüber informieren, was ihr benötigt, um zur Prüfung zugelassen zu werden (z.B. 60\% auf den Übungszetteln) und was genau \emph{eine} Prüfung beinhaltet (z.\,B.\ das Bestehen von einer von zwei Klausuren). Insbesondere der letzte Teil ist wichtig, denn man kann Prüfungen grundsätzlich zweimal schreiben. Je nach Veranstaltung und Dozent/in \emph{können} zwei Klausuren als eine Prüfung zählen, müssen aber nicht -- dann würde jede geschriebene Klausur als ein Prüfungsversuch gelten. Wenn ihr aus Gründen, die ihr nicht selbst verantworten könnt (wie Krankheit), nicht an einer \emph{Prüfung} teilnehmen könnt, erhöht sich entsprechend die Anzahl der Versuche. Wenn ihr alle \emph{Klausur}termine verpasst habt, könnt ihr die Klausur auch durch eine mündliche Prüfung ersetzen. Sollte der Fall auftreten, wendet ihr euch am besten an den/die Dozent/in. 
+
+\subsection{Bachelor-Arbeit}
+Die Bachelor-Arbeit wird im 50\%-Bachelor im ersten Hauptfach geschrieben. Falls du also beispielweise Englisch als erstes Hauptfach und Informatik als zweites Hauptfach hast, musst du dich vor der Anmeldung zur Bachelor-Arbeit umschreiben, quasi deine Fächer tauschen. 
```

Comment:
Gilt natürlich nur, wenn man seine Bachelor-Arbeit in Informatik schreiben will.